fe/fe-core/src/main/java/com/starrocks/catalog/system/information/BeDataCacheMetricsTable.java

```diff
@@ -14,19 +14,35 @@
 
 package com.starrocks.catalog.system.information;
 
+import com.starrocks.catalog.ArrayType;
 import com.starrocks.catalog.PrimitiveType;
 import com.starrocks.catalog.ScalarType;
+import com.starrocks.catalog.StructField;
+import com.starrocks.catalog.StructType;
 import com.starrocks.catalog.Table;
 import com.starrocks.catalog.system.SystemId;
 import com.starrocks.catalog.system.SystemTable;
 import com.starrocks.thrift.TSchemaTableType;
 
+import java.util.ArrayList;
+
 import static com.starrocks.catalog.system.SystemTable.MAX_FIELD_VARCHAR_LENGTH;
 import static com.starrocks.catalog.system.SystemTable.NAME_CHAR_LEN;
 
 public class BeDataCacheMetricsTable {
 
     public static SystemTable create() {
+        ArrayList<StructField> dirSpacesFields = new ArrayList<>();
+        dirSpacesFields.add(new StructField("path", ScalarType.createVarcharType(MAX_FIELD_VARCHAR_LENGTH)));
+        dirSpacesFields.add(new StructField("quota_bytes", ScalarType.createType(PrimitiveType.BIGINT)));
+        StructType dirSpacesType = new StructType(dirSpacesFields);
+        ArrayType dirSpacesArrayType = new ArrayType(dirSpacesType);
+
+        ArrayList<StructField> usedBytesDetailFields = new ArrayList<>(2);
+        usedBytesDetailFields.add(new StructField("priority_0", ScalarType.createType(PrimitiveType.BIGINT)));
+        usedBytesDetailFields.add(new StructField("priority_1", ScalarType.createType(PrimitiveType.BIGINT)));
```

Comment:
The fixed priority names will limit future scalability, such as adding new priority levels.

Reply:
Using Map<INT, BIGINT> instead.